osFiles/macOS/21x - 12.x/21F2092.json

```diff
@@ -11,50 +11,95 @@
     },
     "deviceMap": [
         "Mac14,2",
-        "Mac14,7"
+        "Mac14,7",
+        "MacBook9,1",
+        "MacBook10,1",
+        "MacBookAir7,1",
+        "MacBookAir7,2-2015",
+        "MacBookAir7,2-2017",
+        "MacBookAir8,1",
+        "MacBookAir8,2",
+        "MacBookAir9,1",
+        "MacBookPro11,4",
+        "MacBookPro11,5",
+        "MacBookPro12,1",
+        "MacBookPro13,1",
+        "MacBookPro13,2",
+        "MacBookPro13,3",
+        "MacBookPro14,1",
+        "MacBookPro14,2",
+        "MacBookPro14,3",
+        "MacBookPro15,1-2018",
+        "MacBookPro15,1-2019",
+        "MacBookPro15,2-2018",
+        "MacBookPro15,2-2019",
+        "MacBookPro15,3-2018",
+        "MacBookPro15,3-2019",
+        "MacBookPro15,4",
+        "MacBookPro16,1",
+        "MacBookPro16,2",
+        "MacBookPro16,3",
+        "MacBookPro16,4",
+        "MacPro6,1",
+        "MacPro7,1",
+        "MacPro7,1-Rack",
+        "Macmini7,1",
+        "Macmini8,1",
+        "iMac16,1",
+        "iMac16,2",
+        "iMac16,2-4K",
+        "iMac17,1",
+        "iMac18,1",
+        "iMac18,2",
+        "iMac18,3",
+        "iMac19,1",
+        "iMac19,2",
+        "iMac20,1",
+        "iMac20,2",
+        "iMacPro1,1"
     ],
     "sources": [
         {
-            "type": "installassistant",
+            "type": "ipsw",
             "deviceMap": [
                 "Mac14,2",
                 "Mac14,7"
             ],
             "links": [
                 {
-                    "url": "https://swcdn.apple.com/content/downloads/40/51/012-26510-A_2NSXZSIE75/qexwamj1dibrkxaxfvi0tj8kkhytuano2b/InstallAssistant.pkg",
+                    "url": "https://updates.cdn-apple.com/2022SpringFCS/fullrestores/012-26509/5D021AD7-2AB3-44E4-A094-67F50D26F09E/UniversalMac_12.4_21F2092_Restore.ipsw",
                     "preferred": true,
-                    "active": false
+                    "active": true
+                },
+                {
+                    "url": "http://updates-http.cdn-apple.com/2022SpringFCS/fullrestores/012-26509/5D021AD7-2AB3-44E4-A094-67F50D26F09E/UniversalMac_12.4_21F2092_Restore.ipsw",
+                    "preferred": false,
+                    "active": true
                 }
             ],
             "hashes": {
-                "sha1": "4bc018cad8420d9be64b12fe3c99155a30c3a0df"
+                "sha1": "0c17176fb5d59a0eccb1db5811a7b5b6dbc6a1a3",
+                "sha2-256": "e81de9b2636d004d6e18dbc70bacfbd248850abcf3fad2e439f5c8a92c22fc12"
             },
-            "size": 11676595623
+            "size": 13697592054
         },
         {
-            "type": "ipsw",
+            "type": "installassistant",
             "deviceMap": [
```

Comment:
this looks broken

Reply:
I think the order just got reversed; I did also fix up the root-level deviceMap just now as I noticed that had a full list